lib/code-caller-python.js

```diff
@@ -455,10 +612,10 @@ class PythonCaller {
   }
 
   _logError(msg) {
-    debug(`enter _logError(), state: ${String(this.state)}, uuid: ${this.uuid}`);
+    this.debug('enter _logError()');
     const errData = this._errorData();
     logger.error(msg, errData);
```

Comment:
Calling `logger.error` here is problematic because it dumps unexpected output to `stdout`, and the parent Docker process doesn't know how to interpret it. We should resolve this before merging.